src/hooks/useMovies.tsx

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React, { useState } from "react";
+import { Movie } from "../constants";
+import { movies } from "../mockData";
+
+const useMovies = () => {
+  const [movie, setMovies] = useState<Movie[]>(movies);
+
+  return movies;
+};
+
+export default useMovies;
```

Comment:
For learning custom hooks this hook is okay, but in production I wouldn't create this simple logic in separate hook.

Reply:
Sure, I plan to change it later to get all movies from API. Here, it is created just to meet criteria.